### DISCUSSION TITLE
Support ARC forwarding on noreply senders

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#696] [Feature Requests]
Currently some email headers are modified in order to respect the SPF, DKIM and DMARC. The use of reverse alias also requires modifying some headers. It'd be nice to enable the ARC forwarding on some senders that will never receive emails (i.e. noreply address). 

To get the list of these senders, we can rely on a crowdsourcing approach, i.e. anyone can contribute to this list.

**Comment by developStorm** [comment]
Would love to see this. We can allow users to flag incoming addresses and automatically mark mailboxes with certain number of flags as no-reply, also implement sender address keyword scanning and recognize no-reply by the incoming/outgoing traffic ratio of a particular sender address.
